lte/gateway/c/core/oai/tasks/nas/esm/sap/esm_recv.hpp

```diff
@@ -131,4 +130,3 @@ esm_cause_t esm_recv_deactivate_eps_bearer_context_accept(
 #ifdef __cplusplus
```

Comment:
Can we remove these fun() out of extern C

Reply:
This will be done in a separate PR that I am working on